system-tests/project-fixtures/react/src/App.cy.jsx

```diff
@@ -4,5 +4,5 @@ import { App } from './App'
 
 it('renders hello world', () => {
   mount(<App />)
-  cy.get('h1').contains('Hello World')
```

Comment:
Why are we clicking on hello world?

Reply:
I want to ensure that the component is interactable (i.e. there's no overlay) I added a comment for this: [`fd175c8` (#21599)](https://github.com/cypress-io/cypress/pull/21599/commits/fd175c863f92b811319ae4792a1ee940cc8a3e2f)